scripts/teamcity.sh

```diff
@@ -21,7 +21,10 @@ popd
 cp -r frontend/build/* backend/public
 # Replace the symbolic link we use in dev with the actual file.
 # On Teamcity the JDeb build doesn't seem to follow the symbolic link while packaging, weirdly
-cp frontend/node_modules/pdfjs-dist/build/pdf.worker.min.js backend/public/third-party/pdf.worker.min.js
+# NOTE: On Github actions this seems to work, and in fact it will complain if you try and run the below line
```

Comment:
I can't comment on parts of this file that weren't modified in this PR, but  lines 4-11 in this script are made redundant by `actions/setup-node@v3` in the workflow so they can be removed I think

https://github.com/guardian/giant/blob/958316e75653ba701c2d2da746acb2c245a93c4a/scripts/teamcity.sh#L4-L11